ui/scss/component/_card.scss

```diff
@@ -1,13 +1,13 @@
 .card {
-  background-color: var(--color-card-background);
```

Comment:
### Issues with translucent cards
- Not obvious there is a text field (color same as background).  Worse on white theme.
- In the "1 | 5 | 25 | 100" tab, not clear it is clickable vs grayed out (background looks the same to me)
- For white theme, no border on tabs?

![image](https://user-images.githubusercontent.com/64950861/147680776-239e5931-094c-4350-8194-b88763bfa3d1.png)

![image](https://user-images.githubusercontent.com/64950861/147681369-d992994c-eb2b-4e3f-89b2-9ec51285ee86.png)

Reply:
Updated.
![image](https://user-images.githubusercontent.com/94839278/147683303-469dff8c-3156-4179-83ef-c0d319b02a9c.png)
![image](https://user-images.githubusercontent.com/94839278/147683336-fed137b8-0a21-4002-80c5-c213d33479db.png)